src/Illuminate/Support/Facades/DB.php

```diff
@@ -87,6 +87,7 @@
  * @method static void unsetTransactionManager()
  * @method static bool pretending()
  * @method static array getQueryLog()
+ * @mothod static array getRawQueryLog()
```

Comment:
```suggestion
 * @method static array getRawQueryLog()
```

Reply:
thank you for your review and suggestion !